polling_stations/apps/data_collection/management/commands/import_tameside.py

```diff
@@ -3,23 +3,89 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E08000008"
-    addresses_name = "local.2018-05-03/Version 1/Democracy_Club__03May2018.tsv"
-    stations_name = "local.2018-05-03/Version 1/Democracy_Club__03May2018.tsv"
-    elections = ["local.2018-05-03"]
-    csv_delimiter = "\t"
+    addresses_name = "local.2019-05-02/Version 1/Democracy_Club__02May2019.CSV"
+    stations_name = "local.2019-05-02/Version 1/Democracy_Club__02May2019.CSV"
+    elections = ["local.2019-05-02"]
 
     def address_record_to_dict(self, record):
         uprn = record.property_urn.strip().lstrip("0")
-
-        if record.addressline6 == "OL5 0EX":
-            return None
+        rec = super().address_record_to_dict(record)
+        postcode = record.addressline6
+        #     if record.addressline6 == "OL5 0EX":
+        #         return None
 
         if uprn == "200002851978":
-            rec = super().address_record_to_dict(record)
             rec["postcode"] = "M43 7RZ"
             return rec
 
-        if record.addressline6 == "SK14 1EY":
+        # Clearly wrong but too many to sort out
+        if uprn in [
+            "100012486537",
+            "100012486572",
+            "100012776038",
+            "100012777167",
+            "10014259385",
+            "10090073761",
+            "10090073763",
+            "10090073764",
+            "200004407864",
+            "200004407867",
+            "200004410055",
+            "200004410061",
+            "200004410550",
+            "200004410656",
+            "200004422675",
+            "200004423681",
+        ] or postcode in ["SK14 1EY", "SK14 4XE", "SK14 4EG", "M34 2WS", "OL67FU"]:
```

Comment:
```suggestion
        ] or postcode in ["SK14 1EY", "SK14 4XE", "SK14 4EG", "M34 2WS", "OL6 7FU"]:
```

OL67FU won't match anything

Reply:
Accepted and fixed up